references/sdk.md

```diff
@@ -48,10 +48,21 @@ Create new data sample asset.
     "test_only": bool,
 }
 ```
+The `path` in the data dictionary must point to a directory representing the
+data sample content. Note that the directory can contain multiple files, all the
+directory content will be added to the platform.
+
+If `local` is true, `path` must refer to a directory located on the local
+filesystem. The file content will be transferred to the server through an
+HTTP query, so this mode should be used for relatively small files (<10mo).
+
+If `local` is false, `path` must refer to a directory located on the server
+filesystem. This mode is well suited for all kind of file sizes.
```

Comment:
Could you adapt this comment for the add_data_samples section as well?

Reply:
Yes, thanks.